bool linear_go_home
int32 linear_set_direction
int32 linear_set_speed
bool linear_stop

int32 rack_set_step_number 
bool rack_at_end
bool rack_at_start
bool rack_move_one

uint16 servo1_target
uint16 servo2_target

bool using_linear
bool using_rack
